src/lib/dom-api-mutation-content.html

```diff
@@ -0,0 +1,111 @@
+<!--
+@license
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="settings.html">
+<script>
+(function() {
+  'use strict';
+
+    var DomApi = Polymer.DomApi.ctor;
+    var Settings = Polymer.Settings;
+
+    DomApi.MutationContent = function(domApi) {
+      DomApi.Mutation.call(this, domApi);
+    };
+
+    DomApi.MutationContent.prototype = Object.create(DomApi.Mutation.prototype);
+
+    Polymer.Base.extend(DomApi.MutationContent.prototype, {
+
+      addListener: function(callback, options) {
+        var h = DomApi.Mutation.prototype.addListener.call(this, callback, 
+          options);
+        this._scheduleNotify();

```

Comment:
Super does notifyInitial (no-op since childNodes is empty).

Dirty check approach seems broken for initial?  Consider simplifying initial by constraining when you can observe?